CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ Features:
 
 * [#328](https://github.com/godaddy/tartufo/pull/328) - Buffer issues beyond --buffer-size to a temporary file
 
+v3.0.0 - 15 March 2022
+----------------------
+
+Allow pygit2 dependency <=1.6,>2
+
```

Comment:
```suggestion
Bug Fixes:

* [#330](https://github.com/godaddy/tartufo/pull/330) - Allow newer versions of pygit2 for newer versions of Python
```
This is a bit more of an accurate/holistic description of the change, and also sticks to the surrounding formatting 😄